### ISSUE TITLE
Preview does not work with local android libs

### ISSUE THREAD

**Issue by mojimi** [NativeScript/nativescript-cli#4170]
**Environment**
Provide version numbers for the following components (information can be retrieved by running `tns info` in your project folder or by inspecting the `package.json` of the project):
 - CLI: 5.0.1
 - Cross-platform modules: 5.0.3
 - Android Runtime: 5.0.0
 - iOS Runtime: 5.0.0
 - Plugin(s): nativescript-vue 3.0.0

**Describe the bug**

When using tns preview --hmr on a project with local .jars libs to be loaded, these aren't properly loaded,
giving the error "Cannot read property 'someLibNamespace' of undefined", undefined being "com".

**To Reproduce**

Start a new project using nativescript-vue-cli 3 template
Add android java libs to App_resources/Android/libs
run tns preview --hmr

**Expected behavior**
The project which runs normally on tns run android --hmr should work on tns preview --hmr
**Sample project**

**Additional context**

**Comment by DimitarTachev** [comment]
Hi @mojimi,

The `preview` command is meant to be used when you are getting started with NativeScript and you don't have any Android or iOS local setup. The command is using a message queue and transfers only the files that do not require a new native build (.js, .css, .html etc.). The only available native libraries are the ones built in the predefined set of plugins of the `NativeScript Preview app`.

The updates in the `App_Resources` folder are requiring a new native build and you should get a warning in the preview command output describing that this change will be skipped and your app may not work as expected.

In other words, using a custom native code in the preview command is not possible and when you need some code that requires a native build, you have to [setup your local environment](https://docs.nativescript.org/start/quick-setup#full-setup) and use the `tns run` command.
